src/router.js

```diff
@@ -4,36 +4,34 @@ import { BrowserRouter as Router, Route, Switch, Redirect } from 'react-router-d
 
 // Components
 import AuthService from './utils/AuthService';
-import App from './app/App';
-import Cohort from './cohort/Cohort';
-import Login from './login/Login';
-import Main from './main/Main';
-import Johari from './johari/Johari';
-import NoMatch from './no-match/NoMatch';
-import MyWindow from './my-window/MyWindow';
-import Admin from './admin/Admin';
-import User from './user/User';
+import App from './App/App';
+import Cohort from './Cohort/Cohort';
+import Login from './Login/Login';
+import Main from './Main/Main';
+import Johari from './Johari/Johari';
+import NoMatch from './NoMatch/NoMatch';
+import MyWindow from './MyWindow/MyWindow';
+import Admin from './Admin/Admin';
+import User from './User/User';
 import Axios from 'axios';
 
 // Styles
-import './shared.css';
+import './app.css';
 
 const auth = new AuthService('jH67SpOvPqTg0Jal6m49SCGdECSsFI4L', 'joahriwindow.auth0.com');
 
+const redirect = props => (
```

Comment:
@Tman22 was having a hard time reading this, so I broke out some functions. I'm not positive I handled the spread operator correctly on `...rest`, but nothing broke when I clicked around the app.